docs/spec/literal.rst

```diff
@@ -398,11 +397,9 @@ maintain backwards-compatibility.
 Interactions with generics
 """"""""""""""""""""""""""
 
-Types like ``Literal[3]`` are meant to be just plain old subclasses of
-``int``. This means you can use types like ``Literal[3]`` anywhere
-you could use normal types, such as with generics.
+Literal types are types, and can be used anywhere a type is expected.
```

Comment:
I know what you're trying to say here, and this is definitely an improvement, but this feels tautological 😄

Reply:
It is a tautology, but I still think it's a useful clarification.